tof_read.py

```diff
@@ -4,16 +4,31 @@
 # Print millimeter distance read. Time-of-flight VL53L0X sensor should be
 # connected to the I2C ports (SDA and SCL).
 
+# Your code should always set up all ToF sensors with unique ids (1-15) FIRST,
+# and only then enable them all. Otherwise they will walk over each other
+# on the I2C bus, since they will all have the same address.
+
 class TimeOfFlightRead(SyncedSketch):
 
+    # Set these to the digital inputs connected to each xshut pin
+    self.tof_pin = 20
```

Comment:
Can you change these two lines to not have the self? In the current state python complains that self isn't defined outside an instance method